docs/modules/ROOT/pages/Getting Started.adoc

```diff
@@ -0,0 +1,252 @@
+
+image::image14.png[image,width=480,height=108]
+
+[[context-root]]
+=== Context Root
+
+By default, the name of the WAR file is taken as the context root on which the application is made available.  Using the Application Configuration screen, this value can be changed. 
+
+image::image15.png[Table Description automatically generated with medium confidence,width=624,height=221]
+
+[[internet-accessible-paths]]
+=== Internet-Accessible Paths
+
+Top of Form
+
+While everything served under a specified context root path is accessible to other applications within the namespace, it is possible to restrict which sub-paths of the context root are accessible over the internet.
+
+To set which sub-paths are accessible, enter space-separated patterns in the form of /prefix*. Sub-paths matching this prefix will be available through the domain name of the namespace, while other paths will only be accessible to other applications within the same namespace.
+
+For example, let's assume Context Root of /orders and Internet-Accessible Paths /details/* /cancel/*.
+
+Then requests like 
+
+https://namespace-domain.payara.app/orders/details/000234 
+
+or 
+
+https://namespace-domain.payara.app/orders/cancel/00234 
+
+are directed to application.
+
+Requests to https://namespace-domain.payara.app/orders/place will be handled by the application handling the root namespace.
+
+Other applications within this namespace can reach the handler for /place of this application by accessing http://applicationName/orders/place.
+
+[[database-configuration]]
+=== Database Configuration 
+
+When you upload your application, Payara Cloud looks for the _persistence.xml_ file. If detected, the necessary minimal configuration parameters for a database source for the reference JNDI name can be specified.  
+
+image::image16.png[Application Description automatically generated,width=624,height=242]
+
+If you are using plain JDBC connections instead of the JPA functionality of Jakarta EE, you can
+
+make use of the @DataSourceDefinition or define the data source within the _web.xml_ file. 
+
+____
+@DataSourceDefinition(name="java:global/jdbc/cloud-postgres",  +
+        className="org.postgresql.ds.PGSimpleDataSource",  +
+        serverName = "$\{MPCONFIG=ds_servername}",  +
+        portNumber = 5432,  +
+        databaseName = "$\{MPCONFIG=ds_databasename}",  +
+        user="$\{MPCONFIG=ds_username}",  +
+        password="$\{MPCONFIG=ds_password}"  +
+) 
+____
+
+  
+
+Or <data-source> element in _web.xml_ 
+
+<data-source>  +
```

Comment:
Should this datasource section look like the code in datasourcedefinition above? it's on a gray background and italics.

Reply:
Yep